### PR TITLE
Users/jnorn/myfirstlocalbranch

### PR DESCRIPTION
**Pull request by JNorn**
Updated in class work from today (Nov 12). Thanks!

### CODE REVIEW

File: Ordering/OrderManagement/OrderFramework.cs [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+
+namespace Ordering.BusinessLogic.OrderManagement
+{
+    class OrderFramework:IOrderManager
```
Comment (baez) [inline Ordering/OrderManagement/OrderFramework.cs:11]: Framework is too big of a name for a class --> OrderManager?

File: Ordering/OrderManagement/OrderFramework.cs [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    {
+        public SortedList<Object, int> newOrder = new SortedList<object, int>();
+
+
+        public void AddItem(Object Item, int Quantity)
```
Comment (baez) [inline Ordering/OrderManagement/OrderFramework.cs:16]: item?

File: Ordering/OrderManagement/OrderFramework.cs (hunk 1 above)
Comment (JNorn) [reply Ordering/OrderManagement/OrderFramework.cs:11]: Sure, I can make that change.

File: Ordering/OrderManagement/OrderFramework.cs (hunk 2 above)
Comment (JNorn) [reply Ordering/OrderManagement/OrderFramework.cs:16]: Oswald is writing the code for order item separately, and was going to submit his own pull request for it. Would you rather we combine it first, and submit it all at once?

**Comment by JNorn** [comment]
Closed request, as I've pushed a newer update

**Pull request closed without merging**